src/ai/backend/gateway/server.py

```diff
@@ -331,6 +328,23 @@ def _get_legacy_handler(handler, app, api_version):
                 handler = _get_legacy_handler(r.handler, subapp, version)
                 app.router.add_route(r.method, legacy_path, handler)
 
+    for pkgname in subapp_pkgs:
+        if pidx == 0:
+            log.info('Loading module: %s', pkgname[1:])
+        subapp_mod = importlib.import_module(pkgname, 'ai.backend.gateway')
+        init_subapp(getattr(subapp_mod, 'create_app'))
+
+    app_plugin_entry_prefix = 'backendai_app_v10'
```

Comment:
Though the code uses simply "app" to represent `aiohttp.web.Application`, I think it's better to use `backendai_webapp_v10` here to distinguish this with other types of apps that might be added in the future.
Just a little more clarification. :wink:

Reply:
Thank you for the feedback!

By the way, is there any reason for using "version 10"?